src/index.html

```diff
@@ -1,12 +1,354 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
Misha did the same in one video and said that it would be useful if  you want to add some outer styles for body. But in this task I don't use it.

Anya, could you please tell me, I should fix it?